init.lua

```diff
@@ -10,15 +10,21 @@ local function get_pitch_lift(y)
 	return l
 end
 
-local mouse_controls = minetest.settings:get_bool("glider_mouse_controls", true)
+local mouse_controls = minetest.settings:get_bool("glider.mouse_controls", true)
+local enable_rockets = minetest.settings:get_bool("glider.enable_rockets", true)
+local rocket_delay = tonumber(minetest.settings:get("glider.rocket_delay") or 10)
 
 local on_step = function(self, dtime, moveresult)
-	self.time_from_last_rocket = math.min(self.time_from_last_rocket+dtime,10)
+	self.time_from_last_rocket = math.min(self.time_from_last_rocket+dtime,rocket_delay)
 	local vel = self.object:get_velocity()
 	local speed = self.speed
 	local actual_speed = math.sqrt(vel.x^2+vel.y^2+vel.z^2)
 	local rot = self.object:get_rotation()
 	local driver = minetest.get_player_by_name(self.driver)
+	if not driver then
```

Comment:
IMO this check should be first thing this function does. No need to calculate time etc if there is no driver anyway and the object is removed